tests/recipes/test_alpaca_generate.py

```diff
@@ -13,37 +13,31 @@
 
 from tests.common import TUNE_PATH
 from tests.recipes.common import RECIPE_TESTS_DIR
-
-from tests.recipes.utils import llama2_small_test_ckpt
-from torchtune import models
+from tests.recipes.utils import llama2_test_config
+from tests.test_utils import CKPT_MODEL_PATHS
 
 _CONFIG_PATH = RECIPE_TESTS_DIR / "alpaca_generate_test_config.yaml"
 
-models.small_test_ckpt = llama2_small_test_ckpt
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
 class TestAlpacaGenerateRecipe:
-    def _fetch_ckpt_model_path(self, ckpt) -> str:
-        if ckpt == "small_test_ckpt":
-            return "/tmp/test-artifacts/small-ckpt-01242024"
-        if ckpt == "llama2.llama2_7b":
-            return "/tmp/test-artifacts/llama2-7b-01242024"
-        raise ValueError(f"Unknown ckpt {ckpt}")
-
-    def test_alpaca_generate(self, capsys, pytestconfig, tmpdir, monkeypatch):
-        large_scale = pytestconfig.getoption("--large-scale")
-        ckpt = "llama2.llama2_7b" if large_scale else "small_test_ckpt"
-
+    @pytest.mark.integration_test
+    def test_alpaca_generate(self, tmpdir, monkeypatch):
+        ckpt = "small_test_ckpt_tune"
+        model_checkpoint = CKPT_MODEL_PATHS[ckpt]
         cmd = f"""
         tune alpaca_generate
-            --config {_CONFIG_PATH} \
-            model=torchtune.models.{ckpt} \
-            model_checkpoint={self._fetch_ckpt_model_path(ckpt)} \
+            --config alpaca_generate \
```

Comment:
Not for this PR, but we should probably have this recipe hook into our config system + be generally better supported. Or, just rip it out entirely. cc @RdoubleA @joecummings

Reply:
My vote is for the latter 😃